clicktests/test.discard.js

```diff
@@ -64,7 +64,10 @@ suite.test('Should be possible to discard a created file without warning message
 });
 
 suite.test('Shutdown', function(done) {
-  environment.shutdown(done, true);
+  environment.shutdown(function() {
+    page = webpage.create();
```

Comment:
It's a problem when API call is made and phantomjs is terminated promise gets resolved with an empty object.  It's a very odd behavior I don't see in actual chrome.  But because empty object doesn't have many of the expected properties it break test.  Maybe there is a proper way to refresh the page rather than shuttingdown and reinit we do.